CONTRIBUTING.md

```diff
@@ -2,35 +2,36 @@
 <!-- START doctoc generated TOC please keep comment here to allow auto update -->
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
 
-- [Contributor License Agreements](#contributor-license-agreements)
```

Comment:
Which change do you mean? The table of contents or the one below with EOL?

EOL is already documented

> 6. **For minor releases** Set EOL date for previous release (prior art: https://github.com/kubernetes-sigs/cluster-api/issues/7146) and update Cluster API support and guarantees in CONTRIBUTING.md (prior art: https://github.com/kubernetes-sigs/cluster-api/pull/8308).

Reply:
@sbueringer I meant the table in this file with release/API version/Supported until notes. Because currently, there is no mention of the need for updating this file in the release-tasks.md

Reply:
I'm confused. I'm literally quoting release-tasks.md in https://github.com/kubernetes-sigs/cluster-api/pull/9119/files#r1284500772 :)

Reply:
We just don't literally call out the individual tables I think. Otherwise it's covered

Reply:
@sbueringer I see now. in `release-tasks.md ` looks like we are mentioning it twice: https://github.com/kubernetes-sigs/cluster-api/blob/main/docs/release/release-tasks.md#update-supported-versions and in https://github.com/kubernetes-sigs/cluster-api/blob/main/docs/release/release-tasks.md#repeatedly-cut-a-release number 6 in the list. I wonder if we need it in both places, since we are referring (in `Prior art`) to update same files in both cases

Reply:
The idea was:
* At the beginning of a new cycle:
  * Add the new version that we are now developing to versions.md
* After we cut the minor release:
  * Set the EOL date for the version that is now EOL
  * Add the minor release we just created to the table

Not entirely sure but I think we don't mention it twice, the PRs just have similar names. And in one case we had some cleanup additionally to do in versions.md for some reason

Reply:
done